[ERROR] 2017-05-31 13:06:29,729 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 13:06:29,936 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 13:10:30,260 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 13:10:30,674 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 13:14:30,985 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 13:14:31,158 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 13:18:31,474 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 13:18:31,758 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 13:22:32,077 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 13:22:32,179 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 13:26:32,508 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 13:26:32,670 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 13:30:32,987 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 13:30:33,199 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 13:34:33,502 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 13:34:33,615 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 13:38:33,924 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 13:38:34,082 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 13:42:34,389 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 13:42:34,521 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 13:45:35,775 | /bench-repo/frappe-bench/apps/frappe/frappe/app.py:
Site: site1.local
Form Dict: {
 "cmd": "frappe.core.doctype.user.user.reset_password", 
 "user": "rohannanda30july@gmail.com"
}
Request Error
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/app.py", line 55, in application
    response = frappe.handler.handle()
  File "/bench-repo/frappe-bench/apps/frappe/frappe/handler.py", line 21, in handle
    data = execute_cmd(cmd)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/handler.py", line 52, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 907, in call
    return fn(*args, **newargs)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/core/doctype/user/user.py", line 746, in reset_password
    user.reset_password(send_email=True)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/core/doctype/user/user.py", line 211, in reset_password
    self.password_reset_mail(link)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/core/doctype/user/user.py", line 230, in password_reset_mail
    "templates/emails/password_reset.html", {"link": link}, now=True)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/core/doctype/user/user.py", line 279, in send_login_mail
    delayed=(not now) if now!=None else self.flags.delay_emails, retry=3)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 418, in sendmail
    communication=communication, now=now, read_receipt=read_receipt, is_notification=is_notification)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/email/queue.py", line 61, in send
    email_account = get_outgoing_email_account(True, append_to=reference_doctype)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/email/smtp.py", line 58, in get_outgoing_email_account
    frappe.OutgoingEmailError)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 316, in throw
    msgprint(msg, raise_exception=exc, title=title, indicator='red')
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 306, in msgprint
    _raise_exception()
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 279, in _raise_exception
    raise raise_exception, encode(msg)
OutgoingEmailError: Please setup default Email Account from Setup > Email > Email Account
[ERROR] 2017-05-31 13:45:36,351 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/error.py:
New Exception collected with id: 2017-05-31 13:45:35.777299-127.0.0.1-e06
[ERROR] 2017-05-31 13:46:34,816 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 13:46:34,977 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 13:50:35,287 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 13:50:35,454 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 13:54:35,749 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 13:54:35,918 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 13:58:36,400 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 13:58:36,477 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 14:02:36,795 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 14:02:36,920 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 14:06:37,415 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 14:06:37,484 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 14:10:37,832 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 14:10:38,107 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 14:14:38,399 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 14:14:38,562 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 14:18:38,855 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 14:18:39,026 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 14:22:39,354 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 14:22:39,804 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 14:26:40,128 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 14:26:40,289 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 14:30:40,614 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 14:30:40,773 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 14:34:41,090 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 14:34:41,244 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 14:38:41,548 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 14:38:41,750 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 14:42:42,074 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 14:42:42,238 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 14:46:42,563 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 14:46:42,707 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 14:50:43,022 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 14:50:43,118 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 14:54:43,437 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 14:54:43,785 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 14:58:44,107 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 14:58:44,251 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 15:02:44,578 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 15:02:44,768 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 15:06:45,087 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 15:06:45,525 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 15:10:45,846 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 15:10:46,057 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 15:14:46,377 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 15:14:46,561 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 15:18:46,888 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 15:18:47,050 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 15:22:47,377 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 15:22:47,532 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 15:26:47,864 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 15:26:48,071 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 15:30:48,396 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 15:30:48,767 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 15:34:49,066 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 15:34:49,226 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 15:38:49,523 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 15:38:49,700 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 15:42:49,998 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 15:42:50,151 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 15:46:50,453 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 15:46:50,537 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 15:50:50,845 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 15:50:50,999 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 15:54:51,821 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 15:54:52,196 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 15:58:52,510 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 15:58:52,705 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 16:02:53,029 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 16:02:53,306 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 16:06:53,630 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 16:06:54,043 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 16:10:54,357 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 16:10:54,502 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 16:14:54,799 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 16:14:54,882 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 16:18:55,190 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 16:18:55,287 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 16:22:55,719 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 16:22:55,815 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 16:26:56,134 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 16:26:56,300 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 16:30:56,615 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 16:30:57,368 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 16:34:57,670 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 16:34:57,830 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 16:38:58,127 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 16:38:58,271 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-05-31 16:42:58,562 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-05-31 16:42:58,777 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-06-01 12:41:22,706 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-06-01 12:41:24,652 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-06-01 12:45:24,962 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-06-01 12:45:25,016 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-06-01 12:49:25,336 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-06-01 12:49:25,474 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-06-01 12:53:25,792 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-06-01 12:53:25,967 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-06-01 12:57:26,291 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-06-01 12:57:26,435 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-06-01 13:01:26,749 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-06-01 13:01:26,892 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-06-01 13:05:27,211 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-06-01 13:05:27,348 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-06-01 13:09:27,667 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-06-01 13:09:27,783 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-06-01 13:13:28,111 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-06-01 13:13:28,292 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-06-01 13:17:28,620 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-06-01 13:17:28,752 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-06-01 13:21:29,064 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-06-01 13:21:29,226 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-06-01 13:25:29,543 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-06-01 13:25:29,679 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-06-01 13:29:30,016 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-06-01 13:29:30,143 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-06-02 14:02:01,132 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-06-02 14:02:01,668 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-06-02 14:06:01,977 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-06-02 14:06:02,118 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
[ERROR] 2017-06-02 14:10:02,429 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: royklickinvite.erpnext.com
Exception in Enqueue Events for Site royklickinvite.erpnext.com
Traceback (most recent call last):
  File "/bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py", line 55, in enqueue_events_for_site
    frappe.init(site=site)
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 134, in init
    local.conf = _dict(get_site_config())
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 186, in get_site_config
    config.update(get_file_json(site_config))
  File "/bench-repo/frappe-bench/apps/frappe/frappe/__init__.py", line 864, in get_file_json
    return json.load(f)
  File "/usr/lib/python2.7/json/__init__.py", line 291, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 380, in raw_decode
    obj, end = self.scan_once(s, idx)
ValueError: Expecting property name: line 5 column 1 (char 94)

[DEBUG] 2017-06-02 14:10:02,545 | /bench-repo/frappe-bench/apps/frappe/frappe/utils/scheduler.py:
Site: site1.local
Queued events for site site1.local
